frontend/src/Components/Mentoring.jsx

```diff
@@ -0,0 +1,195 @@
+                            <span className='d-block h3'>{`${volunteer.v_first_name} ${volunteer.v_last_name}`}</span>
+                            <a className='d-block' href={`mailto:${volunteer.v_email}`} target='_blank' rel='noopener noreferrer'>
+                                {volunteer.v_email} 
+                            </a>
+                            <span className='d-block'><strong>Company: </strong>{volunteer.company}</span>
+                            <span className='d-block'><strong>Title: </strong>{volunteer.title}</span>
+                                <ul className='plainUl'> Skills:
+                                    { volunteer.skills
+                                        ? volunteer.skills.map((skill, index) => <li className='ml-2' key={skill+index}>{skill}</li>)
+                                        : null
+                                    }   
+                                </ul>
+                        </div>
+
+                        {/* <div className='col-sm-12'>
+                            <span className='d-block'><strong>LinkedIn: </strong>{volunteer.v_linkedin}</span>
+                            <span className='d-block'><strong>Bio: </strong>{volunteer.v_bio}</span>
+                        </div> */}
+
+                        {/* <div className='col-sm-12 d-flex flex-wrap justify-content-start'>
+                            <strong className='d-block mx-2'>Interested in: </strong>
+                            {
+                                tasks.map((interest, index) => 
+                                    <span key={index + interest[0]} className='d-block mx-2'>
+                                        {interest[0]}
+                                    </span>
+                                )          
+                            }  
+                        </div> */}
+
+                        <div className='col-sm-12'>
+                            <strong className='d-block mx-2'>Mentoring: </strong>
+                            <ul className='plainUl'>
+                                Active:
+                                {
+                                    currentMentees.map(mentee => 
+                                        <li key={mentee[0] + mentee[1] + mentee[2]} className='ml-3'>
+                                            <button className='btn btn-danger btn-sm mr-2 mb-2' onClick={e => deleteMentee(mentee[0])}>X</button>
+                                            <span onClick={e => history.push(`/fellow/${mentee[0]}`)}>
+                                                {mentee[1]}
+                                            </span>: {new Date(mentee[2]).toLocaleDateString()}
+                                        </li>
+                                    )          
+                                }
+                                Ended:
+                                {
+                                    pastMentees.map((mentee, index) => 
```

Comment:
any reason you use the index here to compose the keys while you didnt use it in the currentMentees?

Reply:
It's because in currentMentees the names are unique but in pastMentees there can be duplicate names, I could have used the date but if for some reason a relation gets deleted the same day twice there will be the two children with the same key (it's because the date in the database is just a date without a time). I know there is almost no chance that this happens (except in tests) but just in case